frontend/viewer/src/ProjectView.svelte

```diff
@@ -154,7 +156,14 @@
 </svelte:head>
 
 <div class="project-view !flex flex-col PortalTarget">
-  <AppBar title={projectName} class="bg-secondary min-h-12" menuIcon=''>
+  <AppBar title={projectName} class="bg-secondary min-h-12">
+    {#if showHomeButton}
+      <Button
+        slot="menuIcon"
+        icon={mdiHome}
+        on:click={() => navigate('/')}
```

Comment:
Navigating away (e.g., back to `/`) ends up calling OnDisconnectedAsync on the backend because of SignalR, correct?